crates/adapters/Cargo.toml

```diff
@@ -52,7 +52,7 @@ serde_urlencoded = "0.7.1"
 form_urlencoded = "1.2.0"
 csv = "1.2.2"
 # cmake-build is required on Windows.
-rdkafka = { version = "0.34.0", features = ["cmake-build", "ssl-vendored", "gssapi-vendored"], optional = true }
+rdkafka = { version = "0.37.0", features = ["cmake-build", "ssl-vendored", "gssapi-vendored", "zstd", "libz"], optional = true }
```

Comment:
can we upgrade all the versions we use in all other projects?
Found at least one more. in nexmark

Reply:
good catch, thanks

Reply:
Maybe we should delete the nexmark kafka support. We don't use it anymore.

(Not for this PR though.)